### ISSUE TITLE
invalid request, please check your server timestamp or recv_window when doing  _bybitClient.UsdPerpetualApi.Trading.PlaceOrderAsync

### ISSUE THREAD

**Issue by toshendradalvi** [JKorf/Bybit.Net#30]
When calling _bybitClient.UsdPerpetualApi.Trading.PlaceOrderAsync(..) I get this? is there any way to make this work 100% 
Because intermittent operation pose a threat to stability of the system.

**Comment by toshendradalvi** [comment]
Just need to increase rece_window timeout , I am setting it in constructor 
 _bybitClient = new BybitClient(new BybitClientOptions()
            {
                ApiCredentials = new ApiCredentials(AppConfigurationService.ApiKey, AppConfigurationService.ApiSecret),
                LogLevel = LogLevel.Trace,
                RequestTimeout = TimeSpan.FromSeconds(60),
                ReceiveWindow = new TimeSpan(0,0,30),
            });
but it's still 5000 when doing _bybitClient.UsdPerpetualApi.Trading.PlaceOrderAsync(....)
Not sure why 30 sec is not comes in to play ?

**Comment by JanKorf** [comment]
Hm that sounds like a bug. When setting it like that in the client options it should be send as 30 seconds instead of 5 seconds. Either way, the best way to make sure this error doesn't happen is to make sure your server time is synced more often, see:
https://jkorf.github.io/Binance.Net/Timestamping.html
This is for Binance, but the same goes for Bybit

**Comment by toshendradalvi** [comment]
I think the problem is Parameter used is recvWindow
"&order_type=Market&position_idx=0&qty=0.1&recvWindow=10000&reduce_only=False&side=Sell"
it should be recv_window (underscore missing ?)

**Comment by a-rahmani1998** [comment]
> I think the problem is Parameter used is recvWindow "&order_type=Market&position_idx=0&qty=0.1&recvWindow=10000&reduce_only=False&side=Sell" it should be recv_window (underscore missing ?)

I think you are right, Please update the package @JanKorf

**Comment by JKorf** [comment]
I've looked into this, and it turns out that for the Spot API it's recvWindow while for the futures API's it's recv_window. So that's fun. Either way, I've fixed it in the latest version. 

I've also added the round trip time of a request as a parameter for the timestamp correction, so hopefully there are less timestamp issues from that version on. The best solution for timestamping issue is still to sync your server time more often with a third party tool.
